SpeechResponder/eddi.json

```diff
@@ -42,7 +42,7 @@
       "enabled": true,
       "priority": 3,
       "responder": true,
-      "script": "{_ Context }\r\n{SetState('eddi_context_last_subject', 'body')}\r\n{SetState('eddi_context_last_action', 'scan')}\r\n{SetState('eddi_context_body_system', system.name)}\r\n{SetState('eddi_context_body_name', event.name)}\r\n\r\n{Pause(1000)}\r\n\r\nScan {Occasionally(3,  \"of body\")} {OneOf(\"complete\", \"completed\", \"finished\")}.\r\n\r\n{F(\"Body report\")}\r\n\r\n{if event.estimatedvalue:\r\n   Estimated scan value: {event.estimatedvalue} credits\r\n}",
+      "script": "{_ Context }\r\n{SetState('eddi_context_last_subject', 'body')}\r\n{SetState('eddi_context_last_action', 'scan')}\r\n{SetState('eddi_context_body_system', system.name)}\r\n{SetState('eddi_context_body_name', event.name)}\r\n\r\n{set shortBodyName to event.name}\r\n{set sysNameLen to len(system.name)}\r\n{if len(shortBodyName) > sysNameLen && eq(system.name, slice(shortBodyName, 0, sysNameLen)):\r\n    {set shortBodyName to cat(\"Body\", slice(shortBodyName, sysNameLen))}\r\n}\r\n\r\n{Pause(1000)}\r\n\r\nScan of {P(shortBodyName)} {OneOf(\"complete\", \"completed\", \"finished\")}.\r\n\r\n{F(\"Body report\")}\r\n\r\n{if event.estimatedvalue:\r\n   Estimated scan value: {event.estimatedvalue} credits\r\n}\r\n",
```

Comment:
Add context to save the short name for those who might want to use it in their body report script?
e.g. `{SetState('eddi_context_body_shortname', shortBodyName)}`

Reply:
Done.